src/hb-ot-cmap-table.hh

```diff
@@ -344,7 +344,17 @@ struct CmapSubtableFormat4
       {
 	unsigned int rangeOffset = this->idRangeOffset[i];
 	if (rangeOffset == 0)
-	  out->add_range (this->startCount[i], this->endCount[i]);
+	{
+	  for (hb_codepoint_t codepoint = this->startCount[i];
+	       codepoint <= this->endCount[i];
+	       codepoint++)
+	  {
+	    hb_codepoint_t gid = codepoint + this->idDelta[i];
```

Comment:
I *think* this needs to be modulo 65536?  Check the get_glyph code for it.

Reply:
Right. I did check get_glyph but overlooked this.